app/controllers/pull-requests.js

```diff
@@ -91,8 +91,14 @@ export default class PullRequestsController extends Controller {
 
         return `<a href="${profileLink}" rel="noopener noreferrer" target="_blank">${displayName}</a>`;
       })
-      .join(', ');
 
-    this.contributorsList = contributorsList;
+    this.contributorsList = this.addOxfordComma(contributorsList)
+  }
+
+  addOxfordComma(contributors = []) {
+    if (contributors.length === 2) {
+      return contributors.join(' and ');
+    }
+    return [contributors.slice(0, -1).join(', '), contributors.pop()].filter(contributor => contributor !== '').join(', and ');
```

Comment:
I liked your use of `slice()` to get the first `n - 1` words.

Can we update the code to be general (use the name `words`) and non-mutating (avoid `pop()`)? In this case, we may assume that each word is a non-empty string and skip calling `filter()`.

```suggestion
  addOxfordComma(words = []) {
    if (words.length <= 2) {
      return words.join(' and ');
    }

    return [...words.slice(0, -1), `and ${words[words.length - 1]}`].join(', ');
```